### PR TITLE
User deletion / Technologies documentation / Fix ConfirmMenuItem

### PR DESCRIPTION
**Pull request by yoannfleurydev**

### CODE REVIEW

File: src/app/admin/users/PageUsers.tsx [hunk 1]
```diff
@@ -52,6 +57,7 @@ import {
   PaginationButtonPrevPage,
   PaginationInfo,
 } from '@/components';
+import { ConfirmMenuItem } from '@/components/ConfirmMenuItem';
```
Comment (ivan-dalmet) [inline src/app/admin/users/PageUsers.tsx:60]: use import from `'@/components'` ;)

File: src/app/admin/users/users.service.ts [hunk 2]
```diff
@@ -107,3 +107,15 @@ export const useUserCreate = (
     }
   );
 };
+
+type UserWithLogin = Pick<User, 'login'>;
```
Comment (ivan-dalmet) [inline src/app/admin/users/users.service.ts:111]: Replace with `UserWithOnlyLogin`

**Comment by yoannfleurydev** [comment]
Considering the merge of this one, I think that the PR #26 can be closed

**Review by ivan-dalmet — APPROVED** [review]

File: src/app/admin/users/PageUsers.tsx [hunk 3]
```diff
@@ -91,9 +97,32 @@ const UserActions = ({ user, ...rest }) => {
   const activateUser = () => userUpdate({ ...user, activated: true });
   const deactivateUser = () => userUpdate({ ...user, activated: false });
   const isActionsLoading = userUpdateData.isLoading;
+
+  const queryClient = useQueryClient();
+  const { mutate: userRemove, ...userRemoveData } = useUserRemove({
+    onSuccess: (_, { login }) => {
+      toastSuccess({
+        title: 'Account deleted',
+        description: `Account ${login} deleted with success`,
```
Comment (ivan-dalmet) [inline src/app/admin/users/PageUsers.tsx:106]: If you can add double quotes around the login 😊

File: src/app/admin/users/PageUsers.tsx [hunk 4]
```diff
@@ -91,9 +97,32 @@ const UserActions = ({ user, ...rest }) => {
...
+      });
+      queryClient.invalidateQueries('users');
+    },
+    onError: (_, { login }) => {
+      toastError({
+        title: 'Deletion Failed',
+        description: `Fail to remove ${login} account`,
```
Comment (ivan-dalmet) [inline src/app/admin/users/PageUsers.tsx:113]: If you can add double quotes around the login 😊

**Pull request merged**